2010-05-14  Yavor Doganov  <yavor@gnu.org>

	* Source/ObjectiveC2/GNUmakefile (ADDITIONAL_OBJCFLAGS): Don't
	  pass -march=i686 to GCC; Debian's minimum supported CPU is
	  i486.


--- gnustep-base-1.20.0.orig/Source/ObjectiveC2/GNUmakefile
+++ gnustep-base-1.20.0/Source/ObjectiveC2/GNUmakefile
@@ -51,9 +51,6 @@
 ifeq ($(CC), clang)
   ADDITIONAL_OBJCFLAGS = -fblocks
 endif
-ifeq ($(GNUSTEP_TARGET_CPU), ix86)
-  ADDITIONAL_OBJCFLAGS += -march=i686
-endif
 
 -include Makefile.preamble
 
